Ports/halflife/package.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port="half-life"
+version="1.0.0"
+useconfigure="true"
+depends=("SDL2")
+workdir="xash3d-fwgs-master"
+files="https://github.com/SerenityPorts/xash3d-fwgs/archive/master.tar.gz xash3d_engine.tar.gz
+https://github.com/SerenityPorts/hlsdk-xash3d/archive/master.tar.gz xash3d_hldll.tar.gz"
+launcher_name="Half-Life"
+launcher_category="Games"
+launcher_command="/home/anon/Games/halflife/xash3d"
+
+# This one is a bit tricky to build, so I'm going a little bit off the script....
+configure() {
+    # Configure the shared object projects (client and game)
+    cd ./hlsdk-xash3d-master
+    ./waf configure -T release
+    cd ../
+
+    # Configure the engine itself...
+    cd ./xash3d-fwgs-master
+    ./waf configure --sdl2=../../SDL2/SDL-main-serenity/ -T release
```

Comment:
Does this really require a full source tree?

Reply:
You mean for the port, or the `--sdl` arg?

Reply:
For the `--sdl2` arg. Most other programs/ports are content with whatever is actually installed on the system.

Reply:
This is a weird quirk of `waf`. I've tried massaging it every which way, Nd unless I run this specifically it yells about not being able to find SDL2